### PR TITLE
Fixes: transformers, schema

### PR DESCRIPTION
**Pull request by andreibancioiu**
- Bugfix: fix call of `do_any_extract_task`.
 - Adjust transformers (for _tokens_, for _blocks_).
 - Adjust schema (e.g. `FLOAT64 -> FLOAT`).
 - Update README.

### CODE REVIEW

File: multiversxetl/jobs/transform_job.py [hunk 1]
```diff
@@ -53,6 +53,12 @@ def _do_transform(self, data: Dict[str, Any]) -> Dict[str, Any]:
 class BlocksTransformer(Transformer):
     def _do_transform(self, data: Dict[str, Any]) -> Dict[str, Any]:
         data.pop("pubKeyBitmap", None)
+
+        # Remove "epochStartShardsData.pendingMiniBlockHeaders.reserved".
```
Comment (andreibancioiu) [inline multiversxetl/jobs/transform_job.py:57]: What is the purpose of the `reserved` field?

File: multiversxetl/jobs/transform_job.py [hunk 2]
```diff
@@ -61,6 +67,10 @@ def _do_transform(self, data: Dict[str, Any]) -> Dict[str, Any]:
         data.pop("nft_traitValues", None)
         data.pop("nft_scamInfoType", None)
         data.pop("nft_scamInfoDescription", None)
+        data.pop("nft_scamInfoVersion", None)
+        data.pop("nft_rarity_score", None)
+        data.pop("nft_rarity_rank", None)
+        data.pop("nft_traitTypes", None)
```
Comment (andreibancioiu) [inline multiversxetl/jobs/transform_job.py:73]: Or, perhaps, we should simply remove all fields with the prefix `nft_` instead?

**Review by bogdan-rosianu — APPROVED** [review]

File: multiversxetl/jobs/transform_job.py (hunk 2 above)
Comment (danielailie) [reply multiversxetl/jobs/transform_job.py:73]: we added nft as a prefix to know the service that populate the field, this are added by nft service and when is added by api service we have a prefix api_

File: multiversxetl/jobs/transform_job.py (hunk 1 above)
Comment (miiu96) [reply multiversxetl/jobs/transform_job.py:57]: Is a field from the headers that are used in the protocol, just exposed it also in the ES database.

**Review by miiu96 — APPROVED** [review]

File: multiversxetl/jobs/transform_job.py (hunk 2 above)
Comment (andreibancioiu) [reply multiversxetl/jobs/transform_job.py]: Fixed, made the transformer a bit more generic.

**Review by bogdan-rosianu — APPROVED** [review]

**Review by danielailie — APPROVED** [review]

**Review by miiu96 — APPROVED** [review]

**Pull request merged**